pkgs/top-level/all-packages.nix

```diff
@@ -10229,6 +10229,10 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) CoreFoundation IOKit;
     protobuf = protobuf3_21;
   };
+  netdataCloud = netdata.override {
+    withCloud = true;
```

Comment:
The conditional should go *here*:
```suggestion
    withCloud = !stdenv.isDarwin;
```

Reply:
Or you should guard netdataCloud behind an assert.

Reply:
Thanks, fixed.